packages/x-components/tests/e2e/popular-searches/popular-searches.spec.ts

```diff
@@ -27,13 +27,6 @@ Given(
   }
 );
 
-// Scenario 1
-Then('at most {int} popular searches are displayed', (maxItemsToRender: number) => {
-  cy.getByDataTest('popular-search')
-    .should('have.length.at.least', 1)
-    .and('have.length.at.most', maxItemsToRender);
-});
-
 // Scenario 2
```

Comment:
```suggestion
// Scenario 1
```

Reply:
It is in fact Scenario 2, as Scenario 1 steps are defined in `common-steps.spec.ts`